zer0-os/zOS
# fix connection status in wallet manager

### What does this do?
Fix connection status
### Why are we making this change?

### How do I test this?
- click button connect
- chose metamask
- close metamask popup
- you should see the list of wallet on the wallet-manger
### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
